src/components/dev-hub/hero-banner.stories.mdx

```diff
@@ -0,0 +1,25 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import HeroBanner from './hero-banner';
+import MockAuthorImage from '../../images/1x/MDB-and-Node.js.png';
+
+<Meta title="Hero Banner" />
+
+# Hero Banner
+
+<Preview>
+    <Story name="hero banner">
+        <HeroBanner
+            background={MockAuthorImage}
+            breadcrumb={[
+                { label: 'Home', to: '#' },
+                { label: 'Storybook', to: '#' }
+            ]}
+        >
```

Comment:
```suggestion
        />
```

Reply:
If this is supposed to be closed here, what about the title and byline on lines 18 & 19?

Reply:
Oh my mistake, I think the github grey-out threw me off. Nice catch!

Reply:
Yea no worries. There is some weird stuff going on with this component though so I don't blame you. I think the author byline not showing up though may have to do with the issues I described above. So hopefully updating our config will resolve these discrepancies.